content/en/docs/kubernetes/collector/components.md

```diff
@@ -13,13 +13,148 @@ enhancing it.
 
 Important Components:
 
+- [Kubernetes Attributes Processor](#kubernetes-attributes-processor) - used to
+  add Kubernetes metadata to incoming telemetry.
 - [Filelog Receiver](#filelog-receiver) - used to collect Kubernetes logs and
   application logs written to stdout/stderr.
 
 For application traces, metrics, or logs, we recommend the
 [OTLP receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver),
 but any receiver that fits your data is appropriate.
 
+## Kubernetes Attributes Processor
+
+The Kubernetes Attributes Processor automatically discovers Kubernetes resources
+(pods), extracts their metadata, and adds the extracted metadata to the relevant
+spans, metrics and logs as resource attributes.
+
+**It is one of the most important components for a collector running in
+Kubernetes; any collector receiving application data should use it.** This is
+the processor that adds Kubernetes context to your application traces, metrics,
+and logs. It enables you to correlate your application's traces, metrics, and
+logs with other Kubernetes telemetry such as pod metrics and events.
```

Comment:
```suggestion
**The Kubernetes Attributes Processor is one of the most important
components for a collector running in Kubernetes. Any collector receiving
application data should use it.** Because it adds Kubernetes context to your
telemetry, the Kubernetes Attributes Processor lets you correlate your
application's traces, metrics, and logs signals with our Kubernetes telemetry,
such as pod metrics and traces.
```